assets/js/modules/analytics/datastore/__fixtures__/properties-profiles.json

```diff
@@ -620,6 +620,47 @@
         "href": "https://www.googleapis.com/analytics/v3/management/accounts/151753095/webproperties/UA-151753095-1/profiles/205382010/goals",
         "type": "analytics#goals"
       }
+    },
+    {
+      "accountId": "151753095",
+      "botFilteringEnabled": true,
+      "created": "2019-11-07T11:19:21.809Z",
+      "currency": "USD",
+      "defaultPage": null,
+      "eCommerceTracking": false,
+      "enhancedECommerceTracking": null,
+      "excludeQueryParameters": null,
+      "id": "205382011",
+      "internalWebPropertyId": "214679897",
+      "kind": "analytics#profile",
+      "name": "Some Web Site Data",
+      "selfLink": "https://www.googleapis.com/analytics/v3/management/accounts/151753095/webproperties/UA-151753095-1/profiles/205382011",
+      "siteSearchCategoryParameters": null,
+      "siteSearchQueryParameters": null,
+      "starred": null,
+      "stripSiteSearchCategoryParameters": null,
+      "stripSiteSearchQueryParameters": null,
+      "timezone": "America/Los_Angeles",
+      "type": "WEB",
+      "updated": "2019-11-07T11:19:21.809Z",
+      "webPropertyId": "UA-151753095-1",
```

Comment:
@ivankruchkoff you added an additional profile to the list here but it was referencing a different property ID than the other profile in the list which would be incorrect here. The `properties-profiles` endpoint returns all properties, but the `profiles` returned are always for a single property (either the matched property if there is one, or just the first) – it doesn't return all profiles for each of the properties (that would be way too many requests on the server side). I updated this to be mostly a copy of the other one in the list here with a few differences as this is still needed for the added test 👍